packages/icons/scripts/generate-index.ts

```diff
@@ -0,0 +1,17 @@
+import fs from "fs";
+import type { MultiSourceIconSource } from "./fetch-svgs.ts";
+
+const GENERATED_HEADER = `/*
+* THIS FILE IS GENERATED. DO NOT EDIT IT.
```

Comment:
`This file is generated by the generate-components script. Do not edit directly.`

This will be more consistent with other messages we have in hopper